bounter/tests/cms/test_cms_basic.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Author: Filip Stefanak <f.stefanak@rare-technologies.com>
+# Copyright (C) 2017 Rare Technologies
+#
+# This code is distributed under the terms and conditions
+# from the MIT License (MIT).
+
+import unittest
+
+from bounter import CountMinSketch
+
+
+class MyClass(object):
+    pass
+
+
+class CountMinSketchSanityCommonTest(unittest.TestCase):
+    """
+    Functional tests for setting and retrieving values of the counter
+    """
+
+    def __init__(self, methodName='runTest', algorithm='conservative', delta=0):
+        self.algorithm = algorithm
+        self.delta = delta
+        super(CountMinSketchSanityCommonTest, self).__init__(methodName=methodName)
+
+    def setUp(self):
+        self.cms = CountMinSketch(1, algorithm=self.algorithm)
+
+    def test_unknown_is_zero(self):
+        self.assertEqual(self.cms['foo'], 0)
+
+    def test_increment_default(self):
+        self.cms.increment('foo')
+        self.cms.increment('bar')
+        self.cms.increment('foo')
+        self.cms.increment('foo')
+
+        self.assertEqual(self.cms['foo'], 3)
+        self.assertEqual(self.cms['bar'], 1)
+
+    def test_increment_bytes(self):
+        self.cms.increment('foo')
+        self.cms.increment('bar')
+        self.cms.increment(b'foo')
+        self.cms.increment('foo')
+
+        self.assertEqual(self.cms['foo'], 3)
+        self.assertEqual(self.cms[b'foo'], 3)
```

Comment:
In Python 3, this is likely to cause confusion, IMO.

Python 2:
```python
In [3]: Counter([b'x', u'x']).most_common()
Out[3]: [('x', 2)]
```

Python 3:
```python
In [2]: Counter([b'x', u'x']).most_common()
Out[2]: [(b'x', 1), ('x', 1)]
```

Reply:
Yes. Best to be always explicit with literal strings: either `b''` or `u''`.